src/realm/sort_descriptor.cpp

```diff
@@ -391,6 +402,47 @@ void FilterDescriptor::execute(const Table& table, KeyValues& key_values, const
     key_values = std::move(filtered);
 }
 
+template <typename T, typename U>
+inline int compare(const T& i, const T& j, const U& col)
```

Comment:
template functions are implicitly inline. Is this trying to make it so that the function isn't exported? That should be done by placing it in an anonymous namespace.

Reply:
Fixed, added to anonymous namespace.